src/Components/UI/PostArticle.Styled.js

```diff
@@ -6,5 +6,7 @@ const CardArticle = styled.article`
 	margin: 10px;
 	border-radius: 5px;
 	box-shadow: 2px 2px 8px grey;
+	width: 350px;
+	height: 300px;
```

Comment:
I'd be careful with fixed heights 😉! What if you have a lot of content in your post that exceeds the 300px?